ui: datepicker updates

vladlos opened the pull request:
updated arrow position and controls design

Resolves: #44422, #44893

Release note(ui): none

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/44592)
<!-- Reviewable:end -->

dhartunian commented:
@elkmaster this is great, a big improvement I think!

Can you take a quick look in Firefox? I'm seeing some minor alignment bugs

<img width="333" alt="Screen Shot 2020-02-13 at 12 52 19 PM" src="https://user-images.githubusercontent.com/986307/74463327-af696100-4e5f-11ea-92ea-90337e04b7da.png">

dhartunian commented on pkg/ui/assets/caret-left.tsx:
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+// tslint:disable-next-line: variable-name
+const CuretLeft = () => (
```
Please rename to `Caret`.

dhartunian commented on pkg/ui/assets/caret-left.tsx:
```diff
@@ -0,0 +1,25 @@
+
+// tslint:disable-next-line: variable-name
+const CuretLeft = () => (
+  <svg width={11} height={16} viewBox="0 0 11 16" fill="none">
```
Why do we need to put the svg into a React component? Is there no way to import and use as-is? Does babel have an svg-loader or something? Maybe this? https://webpack.js.org/loaders/svg-inline-loader/

vladlos committed:
ui: datepicker updates

updated arrow position and controls design

Resolves: #44422, #44893

Release note(ui): none

dhartunian commented:
bors r+
